apps/prairielearn/elements/pl-matching/pl-matching.mustache

```diff
@@ -101,4 +101,4 @@
 {{/statements}}
 </div>
 </div>
-{{/answer}}
+{{/answer}}
```

Comment:
Please make sure files finish with a line break.

Reply:
This might be happening because your editor isn't configured to automatically add a new line on file save. You should set this up, as this is a standard convention.

Reply:
Oh okay, thanks! I didn't know this, and got very confused when the current files on master branch didn't have a line break, and the 'make format' command didn't add one either.

I will configure my editor according to the standard convention.